config/ember-try.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-env node */
 module.exports = {
+  useYarn: true,
+  npmOptions: ['--ignore-engines'],
```

Comment:
Why is this required?

Reply:
To run on recent node versions. Ember-cli has `engines` key in package.json, in which they included only `"node": "^4.5 || 6.* || 7.*"`. Because of that yarn fails to install dependencies if node is newer than 7.